rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -267,7 +282,12 @@ def GetBBCords(self, array):
         return b.getBBCords(self._handle, array)
 
     def getImageLabels(self, array):
```

Comment:
Do we still need GetImageLables. If not please remove that from function definitions

Reply:
I wanted to wait for MCW to respond before removing it